Makefile.base

```diff
@@ -44,19 +44,22 @@ OBJ := $(OBJC) $(OBJCXX) $(ASMOBJ) $(ASSMOBJ)
 DEP := $(OBJC:.o=.d) $(OBJCXX:.o=.d) $(ASSMOBJ:.o=.d)
 
 $(BINDIR)$(MODULE)/:
+	@echo "MKDIR $(BINDIR)$(MODULE)" 2>&1
 	$(AD)mkdir -p $@
+#	ls -l -d $(BINDIR)$(MODULE) 2>&1
 
 $(BINDIR)$(MODULE).a $(OBJ): | $(BINDIR)$(MODULE)/
 
 $(BINDIR)$(MODULE).a: $(OBJ) | ${DIRS:%=ALL--%}
+	@echo "ARCHIVE $(BINDIR)$(MODULE)" 2>&1
 	$(AD)$(AR) $(ARFLAGS) $@ $?
 
-
 CXXFLAGS = $(filter-out $(CXXUWFLAGS), $(CFLAGS)) $(CXXEXFLAGS)
 
 # compile and generate dependency info
 
 $(OBJC): $(BINDIR)$(MODULE)/%.o: %.c
```

Comment:
Are any make clean targets called for riot ci? Are all groups being run in
the same tree with a shared output directory? Perhaps there is a race
cleaning up while another group is still building.